RealifeTech-CoreSDK/APILayer/Tests/DiskCacheClearTests.swift

```diff
@@ -12,19 +12,29 @@ import XCTest
 final class DiskCacheClearTests: XCTestCase {
 
     override func setUp() {
+        super.setUp()
         clearObjs()
         populateObjs()
     }
 
     private var objs: [URL] {
-        guard let dir = FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask).first else { return [] }
-        guard let files = try? FileManager.default.contentsOfDirectory(at: dir, includingPropertiesForKeys: nil, options: .skipsHiddenFiles) else { return [] }
+        guard
+            let dir = FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask).first,
+            let files = try? FileManager.default.contentsOfDirectory(
+                at: dir,
+                includingPropertiesForKeys: nil,
+                options: .skipsHiddenFiles)
+        else {
+            return []
+        }
         return files.filter({ $0.absoluteString.contains("\(DiskCache.fileExtension)") })
     }
 
-    private var objNames: [Substring] { return objs.map { $0.absoluteString.split(separator: "$")[1] } }
+    private var objNames: [Substring] {
+        return objs.map { $0.absoluteString.split(separator: "$")[1] }
+    }
 
-    func obj(forURL URL: URL) -> String? {
+    private func obj(forURL URL: URL) -> String? {
         guard let text = try? String(contentsOf: URL.absoluteURL, encoding: .utf8) else { return nil }
```

Comment:
Can we just return this directly without guard:
```
private func obj(forURL URL: URL) -> String? {
    return try? String(contentsOf: URL.absoluteURL, encoding: .utf8)
}
```

Reply:
Sorry that I just saw now. Yes, I'll change it in the next PR